ftw/upgrade/jsonapi/utils.py

```diff
@@ -24,6 +25,14 @@
 import transaction
 
 
+try:
```

Comment:
`IDisableCSRFProtection` was introduced in `plone.protect = 2.x`.
So testing whether `plone.protect` is there is not really helping here, because if it is an `1.x` version it will still fail.

So I could do that:
```python
try:
    pkg_resources.get_distribution('plone.protect')
except pkg_resources.DistributionNotFound:
    DISABLE_CSRF = False
else:
    try:
        from plone.protect.interfaces import IDisableCSRFProtection
    except ImportError:
        DISABLE_CSRF = False
    else:
        DISABLE_CSRF = True
```
but then again...
:wink: